sips/sip-025/sip-025-sbtc.md

```diff
@@ -0,0 +1,907 @@
+participation, but tremendous downside to going offline, which creates a high
+barrier-to-entry for participating as an intermediary.
+
+The system we present, called "sBTC," addresses these two short-comings by
+leveraging unique properties of the Stacks blockchain.  Our key insight is that
+Stacks' PoX consensus algorithm offers a way to _recover from_ 2-way peg safety
+failures while also _properly rewarding_ intermediaries for the value they
+provide.  Specifically:
+
+* We use the fact that the Stacks blockchain "forks with" the Bitcoin
+blockchain to propose an open-membership sBTC wallet maintained by Stackers.
+If Bitcoin forks, then both the Stacker set and their wallet forks with it, and
+the two resulting Stacker sets and wallets each own BTC on their respective
+forks.  This means that Stackers do not lose money from forks, nor do forks
+pose a risk to the safety of users' BTC.
+
+* We use the fact that PoX pays BTC
+to STX holders to propose a liveness recovery mechanism, whereby some BTC
+payouts are repurposed to fulfill withdraw requests should the custodian fail.
+
+* We reward Stackers for staying online to process 2-way peg operations by
+compensating them with BTC via PoX, regardless of the peg's operation volume.  Stackers
+are compensated proportional to their signing power on the peg (i.e how many
+STX they locked) regardless of how many deposit and withdrawal transactions they process.
+
+By leveraging these properties, sBTC offers the following advantages over the
+state-of-the-art:
+
+* **If a majority of Stackers are honest, then sBTC remains safe.**  Every BTC
+withdraw is paired with an equivalent, legitimate request from a wrapped BTC
+holder, _even if every miner is malicious_.  This is achieved by ensuring that
+all Stacking and wallet maintenance operations materialize in all Stacks forks.
+
+* **If a Byzantine fault-tolerant (BFT) majority of miners are honest, then sBTC
+remains live**.  All withdraw requests are eventually fulfilled, _even if every
+Stacker is malicious_.  This is because Stackers do not have a say in block
+production, and Stackers that do not fulfill withdraws lose their PoX-driven BTC
+income from miners.
+
+* **Withdraws of arbitrary amounts of BTC are fulfilled in
+a fixed amount of time on the happy path** if both Stackers and miners operate with a BFT honest
+majority.  If they do not, then withdraws of arbitrary amounts of BTC 
+are fulfilled eventually by _redirecting_ Stackers' PoX payouts to fund withdraw
+requests.
+
+To support this, sBTC offers two modes of operation:
+the Normal mode, and the Recovery mode.  In **Normal mode**, the sBTC asset is
+backed 1:1 by BTC sent to a wallet controlled by a large fraction of Stackers,
+as measured by the fraction of locked STX they represent.  Each time BTC is
+sent to this wallet (a deposit operation), an equal number of sBTC are
+automatically minted to an address of the sender's choosing.  Stackers respond
```

Comment:
More accurate to say "automatically inserted for processing" vs. "automatically minted"? 

The current form makes it sound faultless & guaranteed.